src/pages/cryptoInfo.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import '../assets/cryptoInfo.css';
+
+const CryptoInfo = () => {
+  const cryptoArr = useSelector((state) => state.crypto.cryptoArr);
+
+  return (
+    <div>
+      {cryptoArr ? (
+        <div className="crypto-container-single">
+          {cryptoArr.map((crypto) => (
+            <ul type="button" key={crypto.id} className="single-crypto-info" id={crypto.id}>
+              <li><img src={crypto.image} alt="coinImage" /></li>
+              <li>
+                <h2>
+                  {crypto.name}
+                </h2>
+              </li>
+              <li className="crypto-info-text">
+                <p>
+                  Trading Symbol:
+                </p>
+                <p>
+                  (
+                  {crypto.symbol}
+                  )
+                </p>
+              </li>
```

Comment:
- [ ] The previous reviewer pointed out one key requirement which is to create a search feature where users would search for crypto by name. Kindly work on this feature for the project to get approved. I suggest you create this search bar/area just above the crypto info/items.